File: KahveQRKodOlusturucuJava-calismalarim/app/src/main/java/com/example/qrkodolusturucujava/data/numara_kayit/NumaraKayitSp.java
package com.example.qrkodolusturucujava.data.numara_kayit;

import android.content.Context;
import android.content.SharedPreferences;

public class NumaraKayitSp {

    private SharedPreferences sp;
    private SharedPreferences.Editor editor;

    public NumaraKayitSp(Context context) {
        sp = context.getSharedPreferences("kullanici_bilgileri", Context.MODE_PRIVATE);
        editor = sp.edit();
    }

    public void numaraKeydedildi() {
        editor.putBoolean("kullaniciNumarasiKayitliMi", true).apply();
    }

    public boolean numaraKaydedildiMi() {
        return sp.getBoolean("kullaniciNumarasiKayitliMi", false);
    }

    public String numaraGetir() {
        return sp.getString("numara", "");
    }

    public void numaraKaydet(String kullaniciNumarasi) {
        editor.putString("numara", kullaniciNumarasi).apply();
        numaraKeydedildi();
    }
}

File: KahveQRKodOlusturucuJetpack-calismalarim/app/src/main/java/com/example/qrkodolusturucujetpack/data/numara_kayit/NumaraKayitSp.kt
package com.example.qrkodolusturucujetpack.data.numara_kayit

import android.content.Context
import android.content.Context.MODE_PRIVATE
import javax.inject.Inject


class NumaraKayitSp @Inject constructor(context: Context) {
    val sp = context.getSharedPreferences("kullanici_bilgileri",MODE_PRIVATE)


    val editor = sp.edit()

    fun numaraKeydedildi(){
        editor.putBoolean("kullaniciNumarasiKayitliMi",true).apply()
    }
    fun numaraKaydedildiMi():Boolean{
        return sp.getBoolean("kullaniciNumarasiKayitliMi",false)
    }
    fun numaraGetir():String{
        return sp.getString("numara","")!!
    }
    fun numaraKaydet(kullaniciNumarasi:String){
        editor.putString("numara", kullaniciNumarasi).apply()
        numaraKeydedildi()
    }

}